src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -48,6 +48,10 @@
   <module name="org.openhab.tools.analysis.checkstyle.OverridingParentPomConfigurationCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.RequireBundleCheck">
+      <property name="severity" value="warning" />
```

Comment:
Yes, it is possible to deactivate a check for a bundle. This can happen in the `suppressions.xml` file (https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/suppressions.xml). So, I guess, I will set the severity on "error" and check if some bundle would break the build. If it does, we may discuss whether that bundle should be added to this `suppressions.xml` file